ci.nix

```diff
@@ -42,9 +42,11 @@ dimension "Nixpkgs version" nixpkgsVersions (nixpkgsName: nixpkgs-pin:
       # TODO: can we merge this into the general case by picking an appropriate "cross system" to mean native?
       native = pkgs.recurseIntoAttrs {
         inherit (build) tests maintainer-scripts maintainer-script-cache;
-        hello = (pkgs.haskell-nix.hackage-package { name = "hello"; version = "1.0.0.2"; }).components.exes.hello;
-        iserv-proxy = pkgs.ghc-extra-packages.ghc865.iserv-proxy.components.exes.iserv-proxy;
-        ghc = pkgs.recurseIntoAttrs pkgs.haskell-nix.compiler;
+        sanity-checks = pkgs.recurseIntoAttrs {
+          hello = (pkgs.haskell-nix.hackage-package { name = "hello"; version = "1.0.0.2"; }).components.exes.hello;
+          iserv-proxy = pkgs.ghc-extra-packages.ghc865.iserv-proxy.components.exes.iserv-proxy;
+          ghc = pkgs.recurseIntoAttrs pkgs.haskell-nix.compiler;
+        };
```

Comment:
Moving these one level deeper avoids a strange issue with `release.nix` where it evaluates them when it seems unnecessary (when using ci.nix directly does not).